src/elements/profiles/Profiles.wc.svelte

```diff
@@ -4,7 +4,7 @@
   import type { IFormField, ITab } from "../../types";
   import type { IProfile } from "../../types/Profile";
   import validateMnemonics from "../../utils/validateMnemonics";
-
+  import validateName from "../../utils/validateName";
```

Comment:
this validator is more strict, used for solutions names, unlike in profile manager many people use spaces as part of the profile name e.g `Ahmed Devnet`, let's add another util `validateProfileName` and relax it a bit to include spaces.

Reply:
added a new regex for profile name that includes spaces.